src/pandroid/app/src/main/java/com/panda3ds/pandroid/utils/GameUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.panda3ds.pandroid.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.net.Uri;
+
+import com.google.gson.Gson;
+import com.panda3ds.pandroid.app.GameActivity;
+import com.panda3ds.pandroid.app.PandroidApplication;
+import com.panda3ds.pandroid.data.game.GameMetadata;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class GameUtils {
+    private static final String KEY_GAME_LIST = "gameList";
```

Comment:
Shouldn't this be in `Constants`?

Reply:
this is used only this class, Constants is for global variables